android/src/main/java/com/reactnativewebbrowser/InternalCustomTabsActivitiesHelper.kt

```diff
@@ -54,8 +56,26 @@ class InternalCustomTabsActivitiesHelper(val activityProvider: ActivityProvider)
     return getResolvingActivities(intent).isNotEmpty()
   }
 
-  override fun startCustomTabs(intent: Intent) {
-    currentActivity.startActivity(intent)
+  override fun startCustomTabs(intent: Intent, url: Uri?) {
```

Comment:
intent data already has url right..
no need to pass in as argument

Reply:
updated